Dockerfile

```diff
@@ -37,7 +37,7 @@ WORKDIR /usr/local/src/ort
 
 # Gradle build.
 RUN --mount=type=cache,target=/tmp/.gradle/ \
-    ln -s /tmp/.gradle/ $HOME/.gradle && \
+    GRADLE_USER_HOME=/tmp/.gradle && \
```

Comment:
Mini-nip (I did it wrong myself in the example): Should we end the path with a `/` to highlight it's a directory, and to resemble exactly the path from one line above? Approving anyway.